src/pages/patterns/examples/tables/TableCursorPagination.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Body, Cell, Head, HeaderCell, HeaderRow, Row, Table } from '@zendeskgarden/react-tables';
+import { CursorPagination } from '@zendeskgarden/react-pagination';
+
+const StyledTable = styled(Table)`
+  margin-bottom: ${p => p.theme.space.md};
+  min-width: 500px;
+`;
+
+interface IRow {
+  fruit: string;
+  sun: string;
+  soil: string;
+}
+
+const rowData: IRow[] = Array.from(Array(100)).map((row, index) => ({
+  fruit: `Fruit #${index}`,
```

Comment:
Would it be better to offset the index by +1 and start the count at 1 instead of 0 for demo purposes? Starting a count at 0 feels very "engineering" to me.

Reply:
Well, it is a code example 😛  and the examples come from the production component [page](https://garden.zendesk.com/components/table#pagination). 0 based indices seem okay to me, but feel free to open a new task to change that.